sdk/nodejs/test/index.mjs

```diff
@@ -21,4 +22,36 @@ describe('NodeJS sdk', function () {
           ).then(done());
       });
   });
+
+  it('Connect to engine and execute a simple query to make sure it does not fail', async function () {
+    this.timeout(60000);
+
+    // Use a different port to avoid collision with other tests.
+    const client = await connect({
+      Port: 8081
+    });
+
+    // Close server no matter if the test fails or not.
+    after(async function () {
+      await client.close();
```

Comment:
So this mean we need to call manually the close function to kill the child process right ? 🤔 
If it's the case I would rather call close() implicitly after a Graphql request.

Reply:
Since there's no callback yeah, I could also add a special argument to the "do" function so you can close the client after the query.
Or you could bind a .finally after called do to close the client.

Reply:
@TomChv has more context now and work on another way to handle child process killing

Reply:
Done in https://github.com/dagger/dagger/pull/3652/commits/5d6905c876441a1cae0453ae5402b37aeeb83c24